src/ts/universe.ts

```diff
@@ -326,6 +326,34 @@ export class Asteroid extends UniverseCircle {
   }
 }
 
+export interface UniverseTextOptions extends UniverseObjectOptions {
+  fontSize: number;
+  text: string;
+}
+
+export class UniverseText extends UniverseObject {
+  fontSize: number;
+  text: string;
+
+  constructor(options: UniverseTextOptions) {
+    super(options);
+    this.fontSize = options.fontSize ?? 12;
```

Comment:
This isn't an optional field. Either make it optional in the interface or remove the default.